src/react/WindowVirtualizer.tsx

```diff
@@ -36,6 +37,22 @@ export interface WindowVirtualizerHandle {
    * Get current {@link CacheSnapshot}.
    */
   readonly cache: CacheSnapshot;
+  /**
+   * Scroll to the item specified by index.
+   * @param index index of item
+   * @param opts options
+   */
+  scrollToIndex(index: number, opts?: ScrollToIndexOpts): void;
+  /**
+   * Scroll to the given offset.
+   * @param offset offset from start
+   */
+  scrollTo(offset: number): void;
+  /**
+   * Scroll by the given offset.
+   * @param offset offset from current position
+   */
+  scrollBy(offset: number): void;
```

Comment:
I think scrollTo/scrollBy is not necessary for WindowVirtualizer because you can scroll to the spacific position with `window.scrollTo`/`window.scrollBy`. Do you have any usecases that `WindowVirtualizerHandle.scrollTo/scrollBy` is necessary?

```suggestion
```

Reply:
did this basically just for the sake of thoroughness.